molgenis-metadata-manager/src/main/frontend/src/components/MetadataManagerEntityEditForm.vue

```diff
@@ -72,7 +72,9 @@
         <div class="form-group row">
           <label class="col-4 col-form-label text-muted">{{ 'entity-edit-form-label-attribute-label' | i18n }}</label>
           <div class="col">
-            <multiselect v-model="labelAttribute" :options="attributes" label="label"
+            <multiselect v-model="labelAttribute" @select="selectLabelAttribute"
+                         :options="['CLEAR_SELECTION', ...attributes]"
```

Comment:
what is the label of the 'CLEAR_SELECTION' option ?, is this label translated ? , is it needed as part of the fix ?